source/codegen/templates/service_helpers.mako

```diff
@@ -263,6 +267,8 @@ one_of_case_prefix = f'{namespace_prefix}{function_name}Request::{PascalFieldNam
         response->set_${parameter_name}(${parameter_name});
 %     elif common_helpers.is_struct(parameter) or parameter['type'] == 'ViBoolean[]':
         Copy(${parameter_name}, response->mutable_${parameter_name}());
+%     elif common_helpers.is_bytes_type(parameter['type']):
+        Copy(${parameter_name}, (${underlying_param_type}*)response->mutable_${parameter_name}()->data());
```

Comment:
We probably don't need `->data()` for second parameter. I remember this comment from ViBoolean[] support. Can you remove that and check if it still works (tests pass)?